RELEASE/scripts/autoscend/quests/level_09.ash

```diff
@@ -747,6 +747,7 @@ boolean L9_twinPeak()
 	int starting_trimmers = item_amount($item[Rusty Hedge Trimmers]);
 	if(starting_trimmers > 0)
 	{
+		equipMaximizedGear();
```

Comment:
It was failing the init test without this as it wasn't equipping the required items (Springy Fusilli is the only initiative buff you can use in this path).
This should be changed to use autoAdvBypass though.